virtcontainers/pkg/nsenter/nsenter.go

```diff
@@ -69,6 +63,22 @@ type nsPair struct {
 	threadNS *os.File
 }
 
+func init() {
+	var ns = map[NSType]int{
+		NSTypeCGroup: unix.CLONE_NEWCGROUP,
+		NSTypeIPC:    unix.CLONE_NEWIPC,
+		NSTypeNet:    unix.CLONE_NEWNET,
+		NSTypePID:    unix.CLONE_NEWPID,
+		NSTypeUTS:    unix.CLONE_NEWUTS,
+	}
+
+	for k, v := range ns {
+		if _, err := os.Stat(fmt.Sprint("/proc/self/ns/", string(k))); err == nil {
```

Comment:
I wonder if we could log if some of the namespaces are supported or not. This would be good for future debugging.

Reply:
Yes agree, please add a log statement here.